### ISSUE TITLE
[BUG] [Linter] djlint does not detect unclosed if tag

### ISSUE THREAD

**Issue by alphatownsman** [djlint/djLint#811]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS:  ubuntu 20.04 
 - Python Version (``python --version``) Python 3.11.6
 - djLint Version (``djlint --version``) djlint, version 1.34.1
 - template language: Django

## How To Reproduce
djlint test.html
```
  {% if 1 %}
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
none

**Comment by dapicester** [comment]
See this: https://github.com/djlint/djLint/issues/202#issuecomment-1080618485
